Mods/Alpakit/Source/Alpakit/Private/Alpakit.cpp

```diff
@@ -135,11 +135,13 @@ TSharedRef<FAlpakitProfile> MakeDevelopmentProfileForMod(TSharedRef<IPlugin> Mod
 
     Profile->BuildConfiguration = Settings->GetBuildConfiguration();
 
-    for (auto [CookedPlatform, TargetSetting] : Settings->GetPlatformTargetSettings()) {
+    for (auto& [CookedPlatform, TargetSetting] : Settings->GetPlatformTargetSettings()) {
```

Comment:
Is there a particular reason to make these references?

Reply:
No, I just thought that they didn't need to be copies, which is what I think happens without the &. I defer to you and @mircearoata on this.